BowlingTest.java

```diff
@@ -13,6 +13,7 @@ public void setup() {
 		bowling = new Bowling();
 	}
 
+	// @Ignore
```

Comment:
I deleted these Ignores. Sorry, I was working with outputs of one test for a little bit.